cmd/internal/netmap.go

```diff
@@ -1,4 +1,4 @@
-package common
+package internal
```

Comment:
Usually packages are created in `internal` (which has a special meaning, external software can't import packages from this path), both `internal` and `common` are bad package names on their own. This one deals with netmap and nodes, does pretty-printing for them, maybe `internal/cmdprinter` would be OK.